packages/dds/tree/src/feature-libraries/node-key/nodeKeyManager.ts

```diff
@@ -68,6 +75,16 @@ export function createNodeKeyManager(idCompressor?: IIdCompressor | undefined):
 				assertIsStableId(idCompressor.decompress(extractFromOpaque(key))),
 			);
 		},
+		tryLocalizeNodeKey: (key: string) => {
+			assert(
+				idCompressor !== undefined,
+				0x6e9 /* Runtime IdCompressor must be available to convert node keys */,
+			);
+			const compressedKey = idCompressor.tryRecompress(key as StableNodeKey);
```

Comment:
`key as StableNodeKey`

no cast! cast bad! :D

Use `isStableId()`. IdCompressor only requires that the input is a `StableId`, it doesn't care about `StableNodeKey`.

Reply:
Updated to use `isStableNodeKey` and removed bad cast